src/Portfolio/Portfolio.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Card from "./Card";
+import "./Portfolio.scss";
+
+import portfolioData from "./portfolioData.json";
+import whitestuffLogo from "../../assets/White-Stuff.jpg";
+import FEMpic from "../../assets/FEM.png";
+import sentimentpic from "../../assets/SentimentAnalysis.png";
+
+const Portfolio = () => {
+  return (
+    <div className="portfolio">
+      {portfolioData.map(items => (
+        <Card
+          image={items.image}
+          imageName={items.alt}
+          key={items.name}
+          name={items.name}
+          desc={items.desc}
+        />
+      ))}
+    </div>
+  );
+};
+
+export default Portfolio;
```

Comment:
This is a great use of a generic component where data is passed as props. Well done! 🙌🏼